### PR TITLE
AOI example

### PR DESCRIPTION
**Pull request by philvarner**
**Related Issue(s):** #105 

**Description:**

- add example for AOI calculation

**PR Checklist:**

- [X] Code is formatted
- [X] Tests pass
- [X] Changes are added to the [CHANGELOG](https://github.com/stac-utils/pystac-api-client/blob/main/CHANGELOG.md)

### CODE REVIEW

File: docs/tutorials.rst [hunk 1]
```diff
@@ -19,10 +19,20 @@ STAC Metadata Visualization
...
 
 - :tutorial:`GitHub version <cql2-filter.ipynb>`
 
 This tutorial gives an introduction to using CQL-JSON filtering in searches to
 search by arbitrary STAC Item properties.
+
+Calculating Coverage Percentage of the AOI by an Item
+-----------------------------------------------------
+
+- :tutorial:`GitHub version <aoi-coverage.ipynb>`
```
Comment (gadomski) [inline docs/tutorials.rst:33]:
```suggestion
- :tutorial:`GitHub version <aoi-coverage.ipynb>`
- :ref:`Docs version </tutorials/aoi-coverage.ipynb>`
```

I tried this locally and I _think_ it works, but can you check?

Comment (philvarner) [reply docs/tutorials.rst:33]: are all of them supposed to have that?

Comment (philvarner) [reply docs/tutorials.rst:35]: I added it to all of them, and fixed the ones that were broken. even the one in current docs was broken https://pystac-client.readthedocs.io/en/latest/tutorials/pystac-client-introduction.html -- see the errors

File: docs/tutorials.rst [hunk 2]
```diff
@@ -19,10 +19,21 @@ STAC Metadata Visualization
 This tutorial gives an introduction to using Holeviews and hvplot to visualize
 STAC metadata and Item geometries on a map.
 
-CQL Filtering
+CQL2 Filtering
 ---------------------------
 
 - :tutorial:`GitHub version <cql2-filter.ipynb>`
+- :ref:`Docs version </tutorials/cql2-filter.ipynb>`
```
Comment (philvarner) [inline docs/tutorials.rst:26]: @gadomski I didn't add it to the Metadata Visualization one b/c it needs additional libraries, and PROJ 8 is a pain to install on Ubuntu

File: docs/tutorials/pystac-client-introduction.ipynb [hunk 3]
```diff
@@ -190,7 +188,9 @@
     "# note that this will work in JupyterLab, but not in a Jupyter Notebook\n",
     "\n",
     "import IPython.display\n",
-    "IPython.display.JSON(item_json)"
+    "\n",
+    "\n",
+    "IPython.display.JSON([i.to_dict() for i in items])"
```
Comment (philvarner) [inline docs/tutorials/pystac-client-introduction.ipynb:193]: this was invalid code, so fixed now.

**Review by gadomski — APPROVED** [review]

**Pull request merged**